### ISSUE TITLE
Mac install issues with nfldb-update

### ISSUE THREAD

**Issue by mikemacgregor** [BurntSushi/nfldb#137]
New to python and postgres but I think getting close to proper installation of nfldb on my Mac.

I used Postgress.app (http://postgresapp.com) for the database as that seemed the easiest way to install it on the Mac. I appear to have nfldb installed correctly as I can run the top 10 qb test at the bottom of the install page. When I run nfldb-update however, I get this message:

```
>>> nfldb-update
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'update' is not defined
>>> 
```

There were some hiccups reported during installation (which I've unfortunately since lost from my terminal since it was late last night). Any thoughts as to what might be causing the issue on update?

Thanks.

**Comment by BurntSushi** [comment]
`nfldb-update` is a command line program that you should be running from your shell. It looks like you're trying to run it inside a Python interpreter. (Technically, it's possible to do this, but it's not the recommended way to run `nfldb-update`, and it is more complex.)

**Comment by mikemacgregor** [comment]
Thanks for the quick response. I'm getting confused what should be in the different terminal windows. I looked back at issue #88 and thought maybe psycopg2 should be an older version, as suggested there. Tried that. Then I saw your note. Once I upgraded nfldb:

`
sudo pip install --upgrade nfldb
`

then now it seems to update. I was updating nflgame fine previously but haven't spent a ton of time with these yet so virtually nothing is second nature. Appreciate the work. Cheers.
